synapse/lib/agenda.py

```diff
@@ -305,17 +306,18 @@ class _Appt:
     Each such entry has a list of ApptRecs.  Each time the appointment is scheduled, the nexttime of the appointment is
     the lowest nexttime of all its ApptRecs.
     '''
-    def __init__(self, stor, iden, recur, indx, query, creator, recs, nexttime=None):
+    def __init__(self, stor, iden, recur, indx, query, creator, recs, nexttime=None, view=None):
         self.doc = ''
         self.name = ''
         self.stor = stor
         self.iden = iden
-        self.recur = recur # does this appointment repeat
+        self.recur = recur  # does this appointment repeat
         self.indx = indx  # incremented for each appt added ever.  Used for nexttime tiebreaking for stable ordering
         self.query = query  # query to run
-        self.creator = creator # user iden to run query as
+        self.creator = creator  # user iden to run query as
         self.recs = recs  # List[ApptRec]  list of the individual entries to calculate next time from
-        self._recidxnexttime = None # index of rec who is up next
+        self._recidxnexttime = None  # index of rec who is up next
```

Comment:
Are the above formatting edits part of some some python spec that we should be standardizing on?

Reply:
Oh, that was just me on autopilot getting flake8 to be quiet.